src/libs/sfdk/vboxvirtualmachine.cpp

```diff
@@ -429,6 +428,21 @@ void VBoxVirtualMachinePrivate::setVideoMode(const QSize &size, int depth,
         setExtraData(key, value, Sdk::instance(), [=](bool ok) { *allOk &= ok; });
     };
 
+    {
+        BatchComposer composer = BatchComposer::createBatch("setLastNormalWindowPosition");
+        fetchExtraData("GUI/LastNormalWindowPosition", Sdk::instance(), [=, batch = composer.batch()](QString data, bool ok) {
```

Comment:
Too long, I would prefer starting the lambda on next line.

Reply:
fixed